vercel.json

```diff
@@ -1739,10 +1739,6 @@
       "source": "/tctl-next/workflow#reset",
       "destination": "/cli/workflow#reset"
     },
-    {
-      "source": "/tctl-next/modifiers#--event-id",
```

Comment:
I would not recommend messing with any `tctl` redirects at this time. `tctl` was our workhorse for years and many sources still link in.

Reply:
got it, any objection to pointing it to the broad `/ci/index` page then? There doesn't seem to be a better replacement path for this that I can see

Reply:
Linking to /cli/cmd-options#event-id is semantically connected to what that inbound link is looking for. It gets them to the right revised place. I'm going to say it really depends on how we re-structure the page for clarity and concision. If there is a semantic equivalent, we should always do that. If that anchor goes away, then we can be less specific if it still gets them close to that information.

Reply:
Feedback from team: "Until we no longer support a feature or product, we’ll keep the documentation around and keep redirects in place as needed." No changes to redirects for now.

Reply:
Got it, since the anchor is going away, I think `/cli` is the closest we can get to this information (although with this link not existing today, it's not clear what information it's trying to link anyways).

I'm also removing a duplicate set of redirects, it looks like the same source URLs occur twice in this file.

Reply:
<img width="755" alt="image" src="https://github.com/user-attachments/assets/e88b5e4c-d353-44ff-955a-4e9235f408d0">

Reply:
That's good to know, but I still think `/cli` is the right redirect here. There is no reference to "modifiers" in the docs, so there's no clear location to set a manual anchor. I think the answer here is to the CLI docs and let the user find what they're looking for.